python/ray/includes/unique_ids.pxi

```diff
@@ -134,8 +134,6 @@ cdef class ObjectID(BaseID):
         self.in_core_worker = False
 
         worker = ray.worker.global_worker
-        # TODO(edoakes): there are dummy object IDs being created in
-        # includes/task.pxi before the core worker is initialized.
```

Comment:
@edoakes can you confirm that it's ok to remove this comment?

Reply:
This comments means we should be able to remove the `hasattr` check below and the related `in_core_worker` flag. Let's either remove those or leave the TODO modified to say those can be removed now.

Reply:
Done. Initially tried removing the code, but ran into some errors, so decided to just update the comment.

Reply:
In particular, ran into the error
>AttributeError: 'Worker' object has no attribute 'core_worker'

here